critiquebrainz/frontend/templates/review/entity/base.html

```diff
@@ -45,6 +45,25 @@ <h3>{{ review_credit(review, user_picture_size=24) }}</h3>
         {% endif %}
       {% endif %}
 
+      {% if current_user.is_authenticated and current_user != review.user and current_user.is_admin() %}
```

Comment:
admins should be able to hide their own reviews as well. It won't be needed in real life, but it'd make testing easier.

Reply:
@paramsingh Ok.